rosdoc2/verbs/build/builders/sphinx_builder.py

```diff
@@ -451,12 +443,17 @@ def generate_default_project_into_directory(self, directory):
             'package_authors': ', '.join(set(
                 [a.name for a in package.authors] + [m.name for m in package.maintainers]
             )),
-            'package_underline': '=' * len(package.name),
         }
 
         with open(os.path.join(directory, 'conf.py'), 'w+') as f:
             f.write(default_conf_py_template.format_map(template_variables))
 
+        root_title = f'Welcome to {package.name} documentation'
```

Comment:
I think this would read better as:

```
Welcome to the documentation for {package.name}
```

Reply:
Agreed. See https://github.com/ros-infrastructure/rosdoc2/pull/33/commits/d94267302c5027101a95109df119f7bb365d37d3.